### PR TITLE
Add code to test crate 'legion'

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes:
- Create project 'cubby' (#11)
  - Update project information
  - Add 'IntelliJ Rust' related files and folders to ignore list
  - Correct license to 'GPL-3.0'
- Add code to test crate 'legion' (#11)
  - Add crate 'legion'

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+    "Jaeyong Sung <jaeyong0201@gmail.com>",
+    "Seungbin Oh <sboh1214@gmail.com>",
+    "Minchurl Shin <minchurl001@gmail.com>"
+]
 edition = "2021"
+description = "Voxel-based MMORPG client using Rust"
+repository = "https://github.com/CubbyTeam/Cubby"
+readme = "README.md"
+keywords = ["game", "mmorpg", "voxel", "client"]
+license = "MIT"
```
Comment (stevehan00) [inline Cargo.toml]: Is "MIT" right, not "GPL-3.0"?

Comment (utilForever) [reply Cargo.toml]: Hmm...? It is GPL-3.0.

**Review by buttercrab — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (utilForever) [reply Cargo.toml]: Are you saw previous commit? Please see latest changes. 👀

Comment (stevehan00) [reply Cargo.toml]: Ah, gotcha!

**Review by stevehan00 — APPROVED** [review]

**Pull request merged**